package/build_manager.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+STAGE_DIR=stage
+
+export CHROME_BIN=/usr/bin/google-chrome
+
+rm -rf admin/target
+pushd admin/webapp
+if [[ $# > 0 ]]; then
+    case $1 in
+        -d)
+        mkdir -p /root/.ivy2
+        ln -s /prebuild/manager/cache /root/.ivy2/cache
+        # ln -s /prebuild/manager/node_modules node_modules
```

Comment:
Is this no longer needed?

Reply:
Actually, this line was commented out a long time ago, and we do have the npm install script below for the installation of node_modules, so I think this line can be removed.